packages/react-positioning/src/createArrowStyles.ts

```diff
@@ -0,0 +1,80 @@
+import type { MakeStylesStyleRule } from '@fluentui/make-styles';
+import type { Theme } from '@fluentui/react-theme';
+
+/**
+ * Helper that creates a makeStyles rule for an arrow element.
+ * For runtime arrow size toggling simply create extra classnames to apply to the arrow element
+ *
+ * ```ts
+ *   makeStyles({
+ *     arrowWithSize: createArrowStyles(5),
+ *
+ *     arrowWithoutSize: createArrowStyles(5),
+ *     mediumArrow: { aspectRatio: 1, width: '4px' }
+ *     smallArrow: { aspectRatio: 1, width: '2px' }
+ *   })
+ *   ...
+ *
+ *   state.arrowWithSize.clasName = styles.arrowWithSize
+ *   state.arrowWithoutSize.className = mergeClases(
+ *     styles.arrowWithoutSize,
+ *     state.smallArrow && styles.smallArrow,
+ *     state.mediumArrow && styles.mediumArrow,
+ *   )
+ * ```
+ *
+ * @param size - dimensions of the square arrow element in pixels.
+ */
+export function createArrowStyles(size?: number): MakeStylesStyleRule<Theme> {
+  return theme => {
+    const arrowHCBorderStyle = `1px solid transparent`;
+    const arrowHCBorder = {
+      borderRight: arrowHCBorderStyle,
+      borderBottom: arrowHCBorderStyle,
+    };
+
+    return {
+      position: 'absolute',
+      background: 'inherit',
+      visibility: 'hidden',
+      zIndex: -1,
+
+      ...(size && { aspectRatio: 1, with: `${size}px` }),
```

Comment:
(nit) put each prop on its own line
```suggestion
      ...(size && { 
        aspectRatio: 1,
        with: `${size}px`,
      }),
```

Reply:
Oh, also it looks like a typo? `with` should be `width`
```suggestion
      ...(size && { 
        aspectRatio: 1,
        width: `${size}px`,
      }),
```